lib/Mail/DMARC/Policy.pm

```diff
@@ -56,6 +56,20 @@ sub parse {
     return bless \%policy, ref $self;    # inherited defaults + overrides
 }
 
+sub stringify {
+    my ( $self ) = @_;
+
+    my %dmarc_record = %{$self};
+    # "v" tag must be the first one
+    my $dmarc_txt = 'v=' . $dmarc_record{v} . '; ';
```

Comment:
In perl, delete returns the value of the hash entry, so you can do this:

```suggestion
    my $dmarc_txt = 'v=' . delete $dmarc_record{v} . '; ';
```

Then you don't have to ignore that entry in the foreach loop.